invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -465,11 +465,9 @@ class Meta:
         unknown = EXCLUDE
 
     id = SanitizedUnicode(required=True)
-    # TODO: replace ".en" with UI serialization
-    title = fields.Raw(attribute="metadata.title.en", dump_only=True)
+    title = fields.Raw(attribute="metadata.title")
```

Comment:
Out of curiosity, why did you remove the `dump_only` option?

Reply:
Mistake, adding back.